core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -53,8 +54,10 @@
 @Path(AlluxioMasterRestServiceHandler.SERVICE_PREFIX)
 @Produces(MediaType.APPLICATION_JSON)
 public final class AlluxioMasterRestServiceHandler {
-  private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
   public static final String SERVICE_PREFIX = "master";
+
+  public static final String GET_INFO = "info";
+
```

Comment:
I prefer to keep it to separate the deprecated endpoints from the GET_INFO endpoint